services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -114,7 +114,7 @@ class MetersSpec extends HealthCheckSpecification {
         expect: "Only the default meters should be present on each switch"
         switches.each { sw ->
             def meters = northbound.getAllMeters(sw.dpId)
-            assert meters.meterEntries.size() == 2
+            assert meters.meterEntries.size() == 3
```

Comment:
As far as I understand this should remain '2' for non-noviflow switch

Reply:
thx, fixed